src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -89,7 +89,24 @@ public BoxAPIConnection(String clientID, String clientSecret, String accessToken
      */
     public BoxAPIConnection(String clientID, String clientSecret, String authCode) {
         this(clientID, clientSecret, null, null);
+        this.authenticate(authCode);
+    }
+
+    /**
+     * Constructs a new BoxAPIConnection.
+     * @param  clientID     the client ID to use when exchanging the auth code for an access token.
+     * @param  clientSecret the client secret to use when exchanging the auth code for an access token.
+     */
+    public BoxAPIConnection(String clientID, String clientSecret) {
+        this(clientID, clientSecret, null, null);
+    }
 
+    /**
+     * Authenticates the API connection by obtaining access and refresh tokens using the auth code that was obtained
+     * from the first half of OAuth.
+     * @param authCode the auth code obtained from the first half of OAuth process.
```

Comment:
Minor typo: "the auth code obtained from the first half of __the__ OAuth process."